apps/webapp/src/clients/penumbra-port.ts

```diff
@@ -13,7 +13,7 @@ declare global {
   }
 }
 
-export const getPenumbraPort = (serviceTypeName: string) => {
+export const getPenumbraPort = async (serviceTypeName: string) => {
```

Comment:
Not super related to this PR, but if this is the canonical way that web apps will be connecting to our extension, we should expose this in our own transport package so consumers don't have to think about this.

Reply:
would like to make package-level distinctions between

* penumbra-developed connectrpc transport
* penumbra-developed connectrpc service adapters
* penumbra package providing a client to penumbra services

and we should talk about the precise organization and locations of those boundaries